clients/client-python/setup.py

```diff
@@ -24,16 +24,16 @@
     with open("README.md") as f:
         long_description = f.read()
 except FileNotFoundError:
-    long_description = "Gravitino Python client"
+    long_description = "Apache Gravitino Python client"
 
 setup(
-    name="gravitino",
-    description="Python lib/client for Gravitino",
-    version="0.6.0.dev",
+    name="apache-gravitino",
+    description="Python lib/client for Apache Gravitino",
+    version="0.7.0.dev0",
```

Comment:
@xunliu please be sure to change the version "0.6.0.dev0".

Reply:
DONE.

Reply:
Did you push the code @xunliu

Reply:
OK.